_blueprints/developers_guide.adoc

```diff
@@ -0,0 +1,17 @@
+= Developer's Guide
+
+== Adding new packages
+
+ALMighty uses https://glide.sh[Glide] for package management.  To
+install a new package, you can follow these steps.  The `<URL>` should
+be replaced with the package URL pointing to Github or any other
+location.
+
+[source, bash]
+glide get <URL>
+glide update
```

Comment:
This step is not needed and in fact does more than what you mentioned. It also updates the versions of **all** packages unless they have sticky versions. This is not what you want when you install one new package. `glide get` updates the `glide.lock` already if that is what you're worried about.